src/commands/kol/leaderboard.ts

```diff
@@ -1,130 +1,148 @@
-import { SlashCommandBuilder } from "@discordjs/builders";
-import { CommandInteraction } from "discord.js";
+import {
+  AutocompleteInteraction,
+  ChatInputCommandInteraction,
+  Interaction,
+  SlashCommandBuilder,
+} from "discord.js";
 
 import { createEmbed } from "../../discord";
 import { SubboardInfo, client } from "../../kol";
 
-const PATH_MAPPINGS = new Map([
-  ["clan", 8],
-  ["clandungeons", 8],
-  ["slimetube", 8],
-  ["slime", 8],
-  ["hobopolis", 8],
-  ["hobo", 8],
-  ["hobop", 8],
-  ["beeshateyou", 9],
-  ["bhy", 9],
-  ["bees", 9],
-  ["wayofthesurprisingfist", 10],
-  ["wotsf", 10],
-  ["fist", 10],
-  ["trendy", 11],
-  ["avatarofboris", 12],
-  ["aob", 12],
-  ["boris", 12],
-  ["bugbearinvasion", 13],
-  ["bugbear", 13],
-  ["zombieslayer", 14],
-  ["zombiemaster", 14],
-  ["zombie", 14],
-  ["classact", 15],
-  ["avatarofjarlsberg", 16],
-  ["jarlsberg", 16],
-  ["jarl", 16],
-  ["aoj", 16],
-  ["big", 17],
-  ["kolhs", 18],
-  ["hs", 18],
-  ["highschool", 18],
-  ["classactii", 19],
-  ["classact2", 19],
-  ["avatarofsneakypete", 20],
-  ["sneakypete", 20],
-  ["pete", 20],
-  ["aosp", 20],
-  ["slowandsteady", 21],
-  ["heavyrains", 22],
-  ["hr", 22],
-  ["picky", 23],
-  ["actuallyedtheundying", 24],
-  ["edtheundying", 24],
-  ["ed", 24],
-  ["undying", 24],
-  ["onecrazyrandomsummer", 25],
-  ["1crazyrandomsummer", 25],
-  ["ocrs", 25],
-  ["1crs", 25],
-  ["communityservice", 26],
-  ["cs", 26],
-  ["hccs", 26],
-  ["batfellow", 27],
-  ["avatarofwestofloathing", 28],
-  ["awol", 28],
-  ["thesource", 29],
-  ["source", 29],
-  ["nuclearautumn", 30],
-  ["na", 30],
-  ["gelatinousnoob", 31],
-  ["noob", 31],
-  ["gnoob", 31],
-  ["licensetoadventure", 32],
-  ["lta", 32],
-  ["liveascendrepeat", 33],
-  ["lar", 33],
-  ["pocketfamiliars", 34],
-  ["pokefam", 34],
-  ["glover", 35],
-  ["disguisesdelimit", 36],
-  ["dd", 36],
-  ["darkgyffte", 37],
-  ["dg", 37],
-  ["vampire", 37],
-  ["vampyre", 37],
-  ["twocrazyrandomsummer", 38],
-  ["2crazyrandomsummer", 38],
-  ["tcrs", 38],
-  ["2crs", 38],
-  ["kingdomofexploathing", 39],
-  ["exploathing", 39],
-  ["pathoftheplumber", 40],
-  ["plumber", 40],
-  ["mario", 40],
-  ["lowkeysummer", 41],
-  ["lks", 41],
-  ["greygoo", 42],
-  ["graygoo", 42],
-  ["goo", 42],
-  ["yourobot", 43],
-  ["robot", 43],
-  ["quantumterrarium", 44],
-  ["qt", 44],
-  ["wildfire", 45],
-  ["greyyou", 46],
-  ["grayyou", 46],
-  ["journeyman", 47],
-  ["dinosaur", 48],
-  ["fallofthedinosaurs", 48],
-  ["fotd", 48],
-  ["dinosaurs", 48],
-  ["crimbo22", 49],
-  ["gratitude", 49],
-  ["standard", 999],
-]);
+const BOARD_MAPPINGS = {
+  "Clan Dungeons": 8,
+  "Bees Hate You": 9,
+  "Way of the Surprising Fist": 10,
+  Trendy: 11,
+  "Avatar of Boris": 12,
+  "Bugbear Invasion": 13,
+  "Zombie Slayer": 14,
+  "Class Act": 15,
+  "Avatar of Jarlsberg": 16,
+  "BIG!": 7,
+  KOLHS: 18,
```

Comment:
the 1-word ones w/o special characters don't need " characters on the mappings right? i assume so but wanted to confirm. part of me almost would prefer "" around all of them so that the style matches.

Reply:
prettier did this too sadly